lib/tasks/migrate_data.rake

```diff
@@ -76,4 +76,17 @@ namespace :tww do
       end
     end
   end
+
+  desc "migrate some test data from quickbooks"
```

Comment:
A helper to pull over some QB transactions for testing. Otherwise the transaction list will be empty.